integration/class-webfinger.php

```diff
@@ -23,49 +23,48 @@ public static function init() {
 	/**
 	 * Add WebFinger discovery links
 	 *
-	 * @param array   $array    the jrd array
-	 * @param string  $resource the WebFinger resource
-	 * @param WP_User $user     the WordPress user
+	 * @param array   $jrd    the jrd array
+	 * @param string  $entity the WebFinger resource
```

Comment:
```suggestion
	 * @param string  $uri the WebFinger resource
```

in the user class you named it `$uri` so I would use that here too, because of consistency.